src/main/java/io/debezium/connector/yugabytedb/YugabyteDBCQLValueConverter.java

```diff
@@ -0,0 +1,1062 @@
+import java.math.BigInteger;
+import java.nio.ByteBuffer;
+import java.nio.CharBuffer;
+import java.nio.charset.Charset;
+import java.nio.charset.StandardCharsets;
+import java.sql.SQLException;
+import java.sql.SQLXML;
+import java.sql.Types;
+import java.time.Instant;
+import java.time.LocalDate;
+import java.time.LocalDateTime;
+import java.time.OffsetDateTime;
+import java.time.OffsetTime;
+import java.time.ZoneId;
+import java.time.ZoneOffset;
+import java.time.ZonedDateTime;
+import java.time.format.DateTimeFormatter;
+import java.time.format.DateTimeFormatterBuilder;
+import java.time.temporal.ChronoField;
+import java.time.temporal.TemporalAdjuster;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.BitSet;
+import java.util.Collections;
+import java.util.List;
+import java.util.Map;
+import java.util.Map.Entry;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+import java.util.Optional;
+import java.util.UUID;
+import java.util.concurrent.TimeUnit;
+import java.util.Base64.Encoder;
+import java.util.Base64;
+import java.util.stream.Collectors;
+
+import io.debezium.util.HexConverter;
+import org.apache.kafka.connect.data.Decimal;
+import org.apache.kafka.connect.data.Field;
+import org.apache.kafka.connect.data.Schema;
+import org.apache.kafka.connect.data.SchemaBuilder;
+import org.apache.kafka.connect.errors.ConnectException;
+import org.postgresql.PGStatement;
+import org.postgresql.geometric.PGpoint;
+import org.postgresql.util.HStoreConverter;
+import org.postgresql.util.PGInterval;
+import org.postgresql.util.PGobject;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import com.datastax.driver.core.Duration;
+import com.fasterxml.jackson.core.JsonFactory;
+import com.fasterxml.jackson.core.JsonGenerator;
+import com.google.protobuf.ByteString;
+
+import io.debezium.config.CommonConnectorConfig.BinaryHandlingMode;
+import io.debezium.connector.yugabytedb.YugabyteDBConnectorConfig.HStoreHandlingMode;
+import io.debezium.connector.yugabytedb.YugabyteDBConnectorConfig.IntervalHandlingMode;
+
+import static io.debezium.util.NumberConversions.SHORT_FALSE;
+import static io.debezium.util.NumberConversions.BYTE_ZERO;
+
+import io.debezium.connector.yugabytedb.data.Ltree;
+import io.debezium.data.Json;
+import io.debezium.data.SpecialValueDecimal;
+import io.debezium.data.Uuid;
+import io.debezium.data.VariableScaleDecimal;
+import io.debezium.data.geometry.Geography;
+import io.debezium.data.geometry.Geometry;
+import io.debezium.data.geometry.Point;
+import io.debezium.jdbc.JdbcValueConverters.DecimalMode;
+import io.debezium.jdbc.ResultReceiver;
+import io.debezium.jdbc.TemporalPrecisionMode;
+import io.debezium.jdbc.ValueConversionCallback;
+import io.debezium.relational.Column;
+import io.debezium.relational.ValueConverter;
+import io.debezium.relational.ValueConverterProvider;
+import io.debezium.time.*;
+import io.debezium.util.NumberConversions;
+import io.debezium.util.Strings;
```

Comment:
Review all these imports once, I believe we are not using all of them, for example the postgres ones.